cumulusci/tasks/bulkdata/tests/utils.py

```diff
@@ -7,13 +7,18 @@
 )
 from cumulusci.tests import util as cci_test_utils
 
+current_sf_version = "48.0"
```

Comment:
Can we call this CURRENT_SF_VERSION to match our usual convention for names that are used like constants?

Reply:
And actually, for maximum clarity, how about CURRENT_SF_API_VERSION to be clear it's not a release number?

Reply:
@davisagli Renamed to CURRENT_SF_API_VERSION